src/Glue.elm

```diff
@@ -69,19 +69,7 @@ type Glue model subModel msg subMsg a
 {-| Simple [`Glue`](#Glue) constructor.
 
 Generally useful for composing independent TEA modules together.
-If your module's API is polymofphic use [`poly`](#poly) constructor instead.
-
-**Interface:**
```

Comment:
No point of copy-pasting the **Interface** to the docs, as the full type is [rendered in the docs](https://package.elm-lang.org/packages/turboMaCk/glue/latest/Glue#simple) anyway.

Reply:
It wasn't in some older version :+1: